Sources/APNSKit/apns_kit.swift

```diff
@@ -1,3 +1,29 @@
+import NIOAPNS
+import NIO
+import NIOKit
+
+public final class APNSKitConnectionManager: ConnectionPoolSource {
```

Comment:
I like the name `APNSConnectionSource` for this. That matches what other packages have done so far:

- `RedisConnectionSource`
- `PostgresConnectionSource`
- `MySQLConnectionSource`